### PR TITLE
Support for inactive validator events.

### PR DESCRIPTION
**Pull request by td202**
## Purpose

Closes: #215
Depends on: https://github.com/Concordium/concordium-base/pull/581

Support inactive validator changes.

## Changes

- Update base. This includes updates that support the new `suspended` field on `ConfigureBaker` transactions.
- Add `SpecialTransactionOutcome` cases for `ValidatorSuspended` and `ValidatorPrimedForSuspension`.
- Add `BakerEvent` cases for `BakerSuspended` and `BakerResumed`.
- Add `ValidatorScoreParameters` parameter update effect.
- Rebuild generated protobuf code.

## Checklist

- [X] My code follows the style of this project.
- [x] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [X] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: src/v2/generated/concordium.v2.rs [hunk 1]
```diff
@@ -355,6 +355,14 @@ pub mod account_staking_info {
...
         #[prost(message, optional, tag = "5")]
         pub pool_info:        ::core::option::Option<super::BakerPoolInfo>,
+        /// A flag indicating whether the account is currently suspended or not.
+        /// The flag has a meaning from protocol version 8 onwards. In
+        /// protocol version 8 it signals whether an account has been
+        /// suspended and is not participating in the consensus
+        /// algorithm. For protocol version < 8 the flag will always
+        /// be set to false.
+        #[prost(bool, tag = "6")]
+        pub is_suspended:     bool,
```
Comment (soerenbf) [inline src/v2/generated/concordium.v2.rs:365]: This one does not seem to be reflected in the corresponding type in the SDK, unless I'm missing something?

**Review by soerenbf — APPROVED** [review]
One small thing that seems to be missing, otherwise LGTM! 🎉

File: src/v2/generated/concordium.v2.rs (hunk 1 above)
Comment (td202) [reply src/v2/generated/concordium.v2.rs:365]: Good catch.

**Review by drsk0 — APPROVED** [review]
Looks good!

**Pull request merged**